src/ThrowsAnnotationReader.php

```diff
@@ -130,9 +130,14 @@ private function getDocblock($reflection): ?string
 	{
 		if ($reflection instanceof MethodReflection) {
 			$declaringClass = $reflection->getDeclaringClass();
-			$nativeClassReflection = $declaringClass->getNativeReflection();
-			$nativeMethodReflection = $nativeClassReflection->getMethod($reflection->getName());
-			$docBlock = $nativeMethodReflection->getDocComment();
+			$classReflection = $declaringClass->getNativeReflection();
+			$methodReflection = $classReflection->getMethod($reflection->getName());
+			$docBlock = $methodReflection->getDocComment();
+
+			while ($docBlock === false) {
+				$methodReflection = $methodReflection->getPrototype();
+				$docBlock = $methodReflection->getDocComment();
+			}
 
 			return $docBlock !== false ? $docBlock : null;
```

Comment:
`$docBlock` can't be false in this case.

Reply:
old code, I will fix it

Reply:
I can't remove it, any suggestion?
```
 ------ ---------------------------------------------------------------------- 
  Line   src/ThrowsAnnotationReader.php                                        
 ------ ---------------------------------------------------------------------- 
  140    ReflectionException is never thrown in the corresponding try block    
  147    Method                                                                
         Pepakriz\PHPStanExceptionRules\ThrowsAnnotationReader::getDocblock()  
         should return string|null but returns string|false.                   
 ------ ---------------------------------------------------------------------- 
```

Reply:
Hmm, it's a bug in phpstan :( https://phpstan.org/r/0935e908-80f9-44ec-8de8-de60f11f0a8c

So ok, keep it in this way